sw/airborne/boards/parrot_minidrone/baro_board.c

```diff
@@ -48,37 +50,42 @@ static void *baro_read(void *data __attribute__((unused)))
   struct input_event ev;
   ssize_t n;
 
-  int fd_sonar = open("/dev/input/baro_event", O_RDONLY);
-  if (fd_sonar == -1) {
+  int fd_baro = open("/dev/input/baro_event", O_RDONLY);
+  printf("fd_baro value: %d\n", fd_baro);
+  if (fd_baro == -1) {
     printf("Unable to open baro event to read pressure\n");
     return NULL;
+  } else {
+	printf("Open baro event fd succes\n");
   }
 
   while (TRUE) {
     /* Check new pressure */
-    n = read(fd_sonar, &ev, sizeof(ev));
+    n = read(fd_baro, &ev, sizeof(ev));
     if (n == sizeof(ev) && ev.type == EV_ABS && ev.code == ABS_PRESSURE) {
       pthread_mutex_lock(&baro_swing_mutex);
-      baro_swing_available = true;
+      baro_swing_available = True;
```

Comment:
better stick to `true` (eventually `TRUE`) but not this format which is from Python